selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/FindText.java

```diff
@@ -491,10 +494,12 @@ public void clickOnNextPageButton() {
   }
 
   public Boolean checkNextPageButtonIsEnabled() {
+    WaitUtils.sleepQuietly(1);
```

Comment:
If possible it is better to avoid `WaitUtils.sleepQuietly(1);`

Reply:
`visibilityOfElementLocated` would work well here

Reply:
The **next** and **previous** page buttons are always visible in the **Find Text Info** panel and timeout is needed for waiting that the buttons can change their status after clicking(enabled or disabled).

Reply:
I guess there is a way :)